core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -30,39 +31,83 @@
+
+    public static void withConfig(DistributionStatisticConfig config) {
+        distributionStatisticConfig = config;
+    }
+
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
     /**
-     * Returns a newly-registered {@link DistributionSummary} with percentile publication configured.
-     */
-    public static DistributionSummary summaryWithDefaultQuantiles(MeterRegistry registry,
-                                                                  String name, Iterable<Tag> tags) {
+     * Returns a newly-registered {@link DistributionSummary}.
+     * If distributionStatisticConfig is set, use the config.
+     * If the config is not set, use default percentile.
+     **/
+    public static DistributionSummary defaultDistributionSummary(MeterRegistry registry,
+                                                                 String name, Iterable<Tag> tags) {
         requireNonNull(registry, "registry");
         requireNonNull(name, "name");
         requireNonNull(tags, "tags");
-        return DistributionSummary.builder(name)
-                                  .tags(tags)
-                                  .publishPercentiles(PERCENTILES)
-                                  .register(registry);
+
+        if (distributionStatisticConfig != null) {
+            return DistributionSummary.builder(name)
+                                      .tags(tags)
+                                      .publishPercentiles(distributionStatisticConfig.getPercentiles())
+                                      .publishPercentileHistogram(
+                                              distributionStatisticConfig.isPercentileHistogram())
+                                      .maximumExpectedValue(
+                                              distributionStatisticConfig.getMaximumExpectedValue())
+                                      .minimumExpectedValue(
+                                              distributionStatisticConfig.getMinimumExpectedValue())
+                                      .distributionStatisticBufferLength(
+                                              distributionStatisticConfig.getBufferLength())
+                                      .distributionStatisticExpiry(distributionStatisticConfig.getExpiry())
+                                      .sla(distributionStatisticConfig.getSlaBoundaries())
+                                      .register(registry);
+        } else {
+            return DistributionSummary.builder(name)
+                                      .tags(tags)
+                                      .publishPercentiles(PERCENTILES)
+                                      .register(registry);
+        }
     }
 
     /**
      * Returns a newly-registered {@link Timer} with percentile publication configured.
+     * If distributionStatisticConfig is set, use the config.
+     * If the config is not set, use default percentile.
```

Comment:
`If .... If ....` could be removed and the necessary documentation could be put in the getter and setter of `distributionStatisticConfig`.